run.py

```diff
@@ -15,6 +16,10 @@ def run(
 ) -> None:
 
     for experiment in experiments:
+        overwrite_preprocessing_configs_(experiment.dataloader, staging_config)
+
+        data = experiment.dataloader.load(experiment.dataset_category)
```

Comment:
does this work with multiple datasets? or do we have a plan how it'll work with multiple datasets?